python/tests/test_lowlevel.py

```diff
@@ -166,6 +166,60 @@ class TestTableCollection(LowLevelTestCase):
     Tests for the low-level TableCollection class
     """
 
+    def setUp(self):
+        fd, self.temp_file = tempfile.mkstemp(prefix="msp_ll_tc_")
+        os.close(fd)
+
+    def tearDown(self):
+        os.unlink(self.temp_file)
+
+    @pytest.mark.skipif(IS_WINDOWS, reason="File permissions on Windows")
```

Comment:
I don't see why this needs skipping on windows - we're not touching the file system are we?

Reply:
Again copied without thinking, fixed both in https://github.com/tskit-dev/tskit/pull/986/commits/acda06f0ee1c5a02b13594991b768a0069d6323b